src/test/java/edu/ksu/canvas/attendance/form/CourseConfigurationValidatorUTest.java

```diff
@@ -57,11 +57,11 @@ public void validate_rejectMinutesPerSessionGreaterThanTotalMinutes() {
     public void validate_rejectWrongPoints() {
         courseConfigurationForm.setAssignmentName("Attendance Assignment");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Attendance Assignment" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)